src/components/checkbox/_checkbox.scss

```diff
@@ -179,3 +182,162 @@
     }
   }
 }
+
+@mixin checkbox--experimental {
+  // Spacing between checkboxes
+  .#{$prefix}--form-item.#{$prefix}--checkbox-wrapper {
+    margin-bottom: rem(6px);
+  }
+
+  // Spacing above collection of checkboxes
+  .#{$prefix}--form-item.#{$prefix}--checkbox-wrapper:first-of-type {
+    margin-top: rem(3px);
+  }
+
+  // Spacing below collection of checkboxes
+  .#{$prefix}--form-item.#{$prefix}--checkbox-wrapper:last-of-type {
+    margin-bottom: rem(3px);
+  }
+
+  // Visually, we'll hide the checkbox input and create our own styled version
+  // to match the specs.
+  .#{$prefix}--checkbox {
+    @include hidden;
+  }
+
+  // The label corresponds to the content inside of the `label` tag. Since we're
+  // creating our own checkbox style, we'll need to position this in order to
+  // accomodate the spacing needed for the checkbox.
+  .#{$prefix}--checkbox-label {
+    @include reset;
+    @include font-family;
+    @include font-smoothing;
+    //@include type-scale-item('b');
```

Comment:
Do we need to replicate this mixin in the project?

Reply:
Oh good catch, can we just switch this to our type scale greek letters? Or do we need it mapped to the a,b,c etc?

Reply:
Should we be using the type package? so many little things to think about